### PR TITLE
Doaminconnectmultipleinstanceflag

### PR DESCRIPTION
**Pull request by vishakhasalunkhe-domainconnect**
1. Created separate domain verification template with multiInstance flag set.
 2. Update generic template to remove TXT verification record from it

### CODE REVIEW

File: google.com.verification.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "providerId":"google.com",
+  "providerName":"Google",
+  "serviceId":"verification",
+  "serviceName":"Google GSuite Setup",
```
Comment (navneetgoel-domainconnect) [inline google.com.verification.json]: should be:
"serviceName": "Domain Verification",

File: google.com.verification.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  "version": 1,
+  "logoUrl":"",
+  "description":"Enables a domain to work with Google GSuite",
```
Comment (navneetgoel-domainconnect) [inline google.com.verification.json]: should be:
"Enables a domain to work with Google",

File: google.com.verification.json (hunk 1 above)
Comment (vishakhasalunkhe-domainconnect) [reply google.com.verification.json]: Done

File: google.com.verification.json (hunk 2 above)
Comment (vishakhasalunkhe-domainconnect) [reply google.com.verification.json]: Done

**Pull request closed without merging**